website/StandardAttributes.rst

```diff
@@ -0,0 +1,1388 @@
+         representation (e.g. integer to float) along with scaling of the
+         individual values.
+       </p>
+       <p style="padding-bottom:15px">
+         A common spatial scaling is from a 2880 x 1620 acquisition format to a
+         1920 x 1080 HD format. In this case, a camera solver will want to know
+         the original set of photosites that contributed sensel values to the
+         downscaler: their number, and their position. Through a combination of
+         <tt>sensorAcquisitionRectangle</tt>, <tt>sensorPhotositePitch</tt>,
+         <tt>sensorOverallDimensions</tt> and <tt>sensorCenterOffset</tt>, the application can
+         know exactly the area on the sensor on which the light fell to create
+         the sensel values that produced a monochrome image.
+       </p>
+       <p style="padding-bottom:15px">
+         RGB images are more complicated. RGB pixel reconstruction is a form of
+         filtering, and kernels are square, with relatively small span, e.g. 5x5
+         or 7x7. Edge handling for the kernel is important; the Wikipedia
+         article describing an image processing <a
+         href=https://en.wikipedia.org/wiki/Kernel_(image_processing)> kernel </a> covers it well.
+       </p>
+       <p style="padding-bottom:15px">
+         Elements of the reconstruction kernel that are never at the center of
+         the kernel are <it>not</it> counted as part of the
+         sensorAcquisitionRectangle. Recalling the simple case above of a
+         non-spatially-scaled 2880 x 1620 monochrome image being in 1:1
+         correspondence with an array of photosites on the sensor, if we are
+         instead reading from a CFA-covered sensor to reconstruct a 2880 x 1620
+         RGB image, the actual array of all photosites whose sensel values were
+         fed into a 5x5 reconstruction kernel would not be 2880 x 1620, but 2884
+         x 1624. Nevertheless, the size of the <tt>sensorAcquisitionRectangle</tt> would
+         be 2880 x 1620.
+       </p>
+       <p style="padding-bottom:15px">
+         Camera systems differ on how to handle the case where the position of
+         the RGB reconstruction kernel is such that one or more elements of the
+         kernel do not correspond to physical photosites; these are edge cases
+         in every sense of the phrase.
+       </p>
+     </td>
+   </tr>
+
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> xDensity </b> </tt>
+     </td>
+     <td style="vertical-align: top; width:100px"> <tt> float </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Horizontal output density, in pixels per inch.  The image's vertical
+         output density is <tt> xDensity * pixelAspectRatio</tt>.
+       </p>
+     </td>
+   </tr>
+
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> longitude lattitude altitude </b> </tt>
```

Comment:
typo for latitude?
This might be clearer to do as three separate attribute/type pairs, repeating 'float' each time, and having the definition with a rowspan="3" to make it common to all three. That would also clarify that latitude and longitude are stored as a singe float for degrees, not with minutes and seconds

Reply:
I like that idea, I'll fiddle with it later.

Reply:
I figured out the rowspan, works much better, thanks. I also replaced the other instances where multiple attributes have a grouped definition.